docs/commands/index.md

```diff
@@ -63,6 +63,7 @@ some filenames are provided to replace then only those files will be replaced.
 Theme Kit calculates a checksum for each file, and only updates assets if you've made changes to them locally.
 
 |**Optional Flags**||
+|    |`--allow-live`| Will allow themekit to deploy the file changes to the currently live theme.
```

Comment:
Minor: `Theme Kit` (and similarly below)

Reply:
Aw man, did we finally make a style-guide decision? Well I guess it's better than having the plethora of inconsistencies and grammar issues I always cause lol.

Reply:
I believe @gfscott confirmed it but I can't remember where.

Reply:
I believe I issued an edict one one of Carmela's PRs in the App CLI repo, but now I _also_ can't remember where. I've definitely been using `Theme Kit` — mostly just because that's what the biggest type on the page says?
![Capital T space Capital K](https://screenshot.click/20-09-p9qei-4xidv.png)

But, like, we can decide to change it!